### PR TITLE
Add ligand network output to CLI network planners

### PR DESCRIPTION
**Pull request by dwhswenson**
This adds output of the `LigandNetwork` to the file `$OUTPUT_DIR/ligand_network.graphml` in the `plan-*-network` CLI commands.

I also slightly re-nested, re-adding the `transformations/` dir that was removed in #382.

### CODE REVIEW

**Comment by dwhswenson** [comment]
This should be ready for review.

File: openfecli/commands/plan_rbfe_network.py [hunk 1]
```diff
@@ -57,8 +54,7 @@ def plan_rbfe_network_main(
     alchemical_network = network_planner(
         ligands=small_molecules, solvent=solvent, protein=protein
     )
-
-    return alchemical_network
+    return alchemical_network, network_planner._ligand_network
```
Comment (IAlibay) [inline openfecli/commands/plan_rbfe_network.py:57]: update the return docstring?

File: openfecli/commands/plan_rbfe_network.py [hunk 2]
```diff
@@ -78,7 +74,7 @@ def plan_rbfe_network_main(
 @MAPPER.parameter(required=False, default="LomapAtomMapper")
 @print_duration
 def plan_rbfe_network(
-    molecules: List[str], protein: str, output_dir: str, mapper: str
+    molecules: List[str], protein: str, output_dir: str, mapper: str,
```
Comment (IAlibay) [inline openfecli/commands/plan_rbfe_network.py:77]: I'll update the docstring for this in #401

File: openfecli/commands/plan_rhfe_network.py [hunk 3]
```diff
@@ -47,7 +50,7 @@ def plan_rhfe_network_main(
         ligands=small_molecules, solvent=solvent
     )
 
-    return alchemical_network
+    return alchemical_network, network_planner._ligand_network
```
Comment (IAlibay) [inline openfecli/commands/plan_rhfe_network.py:53]: Update return docstring

File: openfecli/plan_alchemical_networks_utils.py [hunk 4]
```diff
@@ -1,14 +1,18 @@
 # This code is part of OpenFE and is licensed under the MIT license.
 # For details, see https://github.com/OpenFreeEnergy/openfe
+from __future__ import annotations
 import os
 from collections import defaultdict
 import json
 import pathlib
 from openfecli.utils import write
+import typing
```
Comment (IAlibay) [inline openfecli/plan_alchemical_networks_utils.py:10]:
```suggestion


```

**Review by IAlibay — APPROVED** [review]
Just two documentation things, rest seems fine.

**Pull request merged**